packages/@react-spectrum/combobox/stories/ComboBox.stories.tsx

```diff
@@ -0,0 +1,410 @@
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {action} from '@storybook/addon-actions';
+import {Button} from '@react-spectrum/button';
+import {ButtonGroup} from '@react-spectrum/buttongroup';
+import {ComboBox, Item, Section} from '../';
+import Copy from '@spectrum-icons/workflow/Copy';
+import {Flex} from '@react-spectrum/layout';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {Text} from '@react-spectrum/text';
+
+let items = [
+  {name: 'Aardvark', id: '1'},
+  {name: 'Kangaroo', id: '2'},
+  {name: 'Snake', id: '3'}
+];
+
+let withSection = [
+  {name: 'Animals', id: 's1', children: [
+    {name: 'Aardvark', id: '1'},
+    {name: 'Kangaroo', id: '2'},
+    {name: 'Snake', id: '3'}
+  ]},
+  {name: 'People', id: 's2', children: [
+    {name: 'Danni', id: '4'},
+    {name: 'Devon', id: '5'},
+    {name: 'Ross', id: '6'}
+  ]}
+];
+
+storiesOf('ComboBox', module)
+  .add(
+    'no items',
+    () => (
+      <ComboBox items={[]} label="Combobox" onOpenChange={action('onOpenChange')} onInputChange={action('onInputChange')} onSelectionChange={action('onSelectionChange')} onBlur={action('onBlur')} onFocus={action('onFocus')} onCustomValue={action('onCustomValue')}>
+        {(item: any) => <Item>{item.name}</Item>}
+      </ComboBox>
+    )
+  )
+  .add(
+    'static items',
+    () => render({})
+  )
+  // TODO: figure out why item needs a :any here
+  .add(
+    'dynamic items',
+    () => (
+      <ComboBox items={items} label="Combobox" onOpenChange={action('onOpenChange')} onInputChange={action('onInputChange')} onSelectionChange={action('onSelectionChange')} onBlur={action('onBlur')} onFocus={action('onFocus')} onCustomValue={action('onCustomValue')}>
+        {(item: any) => <Item>{item.name}</Item>}
+      </ComboBox>
+    )
+  )
+  // TODO: figure out why item needs a :any here
+  .add(
+    'with sections',
+    () => (
+      <ComboBox items={withSection} label="Combobox" onOpenChange={action('onOpenChange')} onInputChange={action('onInputChange')} onSelectionChange={action('onSelectionChange')} onBlur={action('onBlur')} onFocus={action('onFocus')} onCustomValue={action('onCustomValue')}>
+        {(item: any) => (
+          <Section key={item.name} items={item.children} title={item.name}>
+            {(item: any) => <Item key={item.name}>{item.name}</Item>}
+          </Section>
+        )}
+      </ComboBox>
+    )
+  )
+  .add(
+    'user provided id and label',
```

Comment:
This example is more to demonstrate that the user can use their own label element rather than the built in label prop. The items are static though yes